seqr/views/apis/project_categories_api_tests.py

```diff
@@ -4,15 +4,21 @@
 from seqr.views.apis.project_categories_api import update_project_categories_handler
 from seqr.views.utils.test_utils import _check_login
 
+PROJECT_GUID = 'R0001_1kg'
+
 
 class ProjectCategoriesAPITest(TestCase):
     fixtures = ['users', '1kg_project']
 
     def test_project_categories_api(self):
-        url = reverse(update_project_categories_handler, args=['R0001_1kg'])
+        url = reverse(update_project_categories_handler, args=[PROJECT_GUID])
         _check_login(self, url)
 
         response = self.client.post(url, content_type='application/json', data=json.dumps({
-            'categories': []
```

Comment:
This was actually testing that sending an empty list removes all categories. Please leave this test in, check the response to confirm that the project categories are all removed, and then add the test you have here to add them back

Reply:
Updated.